src/pdm/cli/commands/python.py

```diff
@@ -97,7 +108,20 @@ def handle(self, project: Project, options: Namespace) -> None:
             for version in PYTHON_VERSIONS:
                 project.core.ui.echo(str(version))
             return
-        self.install_python(project, options.version)
+        version = options.version
+        if version is None:
+            match = (
+                project.get_min_matching_cpython_version()
+                if options.min
+                else project.get_best_matching_cpython_version()
```

Comment:
It's not worth for two separate methods, just merge them into one and switch via an argument.

Reply:
done